src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -502,6 +502,14 @@ export default {
 			return !this.isLastMessage && this.id === this.$store.getters.getVisualLastReadMessageId(this.token)
 		},
 
+		renderedMessage() {
+			if (this.messageParameters?.file && this.message !== '{file}') {
+				return `{file} ${this.message}`
```

Comment:
space? Or should be a new line or something?

Reply:
new line `\n\n` would be better, as it puts files and caption in different paragraphs
Although it creates a gap under the image, and doesn't look good with one-liners, we could stylize them later